conf/defaults.ini

```diff
@@ -1105,6 +1105,26 @@ min_interval = 10s
 # (concurrent queries per rule disabled).
 max_state_save_concurrency = 1
 
+[unified_alerting.external_alertmanager]
+
+# Disable the internal Alertmanager and use an external one instead.
+# The default value is `false`.
+disable_internal_alertmanager = false
+
+# URL of the external Alertmanager that will replace the internal one.
+# Required if `disable_internal_alertmanager` is set to `true`.
+main_alertmanager_url =
+
+# Optional tenant ID to attach to requests sent to the Alertmanager.
+main_alertmanager_tenant_id = 
+
+# Optional username for basic authentication on requests sent to the external Alertmanager.
+# Can be left blank to disable basic auth.
+main_alertmanager_basic_auth_username =
```

Comment:
```suggestion
```

Let's use the the tenant. If there's no password set then X-Scope-OrgID is set as a header. Can you please document it accordingly?

Reply:
For ASH we did separate tenant and basic auth user fields. IMO it's confusing to have a basic auth pass without a user.

If it's in OSS, we don't have a lot of guarantees that the user's reverse proxy is set up in a sane way